tasks.py

```diff
@@ -22,6 +24,11 @@ def genspider(ctx, name, domain):
     test_filename = _gen_tests(name, domain)
     print('Created {0}'.format(test_filename))
 
+    if start_urls:
+        html_filenames = _gen_html(name, start_urls)
```

Comment:
Thanks for catching that. Looks like invoke doesn't take lists as arguments yet. I changed genspider to receive a { separated string of urls. I don't know if { is the best choice--just tried to pick a character that doesn't appear much in urls.